vocab/properties.json

```diff
@@ -9168,7 +9766,9 @@
     },
     "description": null,
     "label": "Status",
+    "labelPlural": "Status",
```

Comment:
"Status" is not wrong as a plural, but I think "statuses" is more commonly used and, in this context, communicates more clearly that more than one status value is possible.